app/http/httpclient.c

```diff
@@ -451,20 +451,33 @@ static void ICACHE_FLASH_ATTR http_timeout_callback( void *arg )
 	struct espconn * conn = (struct espconn *) arg;
 	if ( conn == NULL )
 	{
+		HTTPCLIENT_DEBUG( "Connection is NULL" );
 		return;
 	}
 	if ( conn->reverse == NULL )
 	{
+		HTTPCLIENT_DEBUG( "reverse is NULL" );
```

Comment:
It may be helpful to print a more explanatory message. The meaning of "reverse" may not be totally obvious.